tattoo_shop/__manifest__.py

```diff
@@ -0,0 +1,67 @@
+{
+    'name': 'Tattoo Shop',
+    'version': '1.0',
+    'category': 'Services',
+    'author': 'Odoo S.A.',
+    'depends': [
+        'appointment_account_payment',
+        'base_automation',
+        'crm_enterprise',
+        'knowledge',
+        'sale_planning',
+        'sale_purchase',
+        'sale_service',
+        'sign',
+        'social',
+        'web_studio',
+        'website'
```

Comment:
please apply everywhere it makes sense
```suggestion
        'website',
```